neqo-transport/src/connection/tests/resumption.rs

```diff
@@ -107,16 +107,27 @@ fn two_tickets() {
     server.send_ticket(now(), &[]).expect("send ticket2");
     let pkt = send_something(&mut server, now());
 
-    client.process_input(pkt, now());
-    // Increase the current time with a large number to trigger
-    // the resumption_token_timer before calling get_tokens.
-    let mut now = now() + Duration::from_millis(140);
-    let token1 = get_tokens(&mut client, Some(now)).pop().unwrap();
-    now += Duration::from_millis(140);
-    let token2 = get_tokens(&mut client, Some(now)).pop().unwrap();
-    // There is no more tokens.
-    now += Duration::from_millis(140);
-    assert_eq!(get_tokens(&mut client, Some(now)).len(), 0);
+    // process() will return a ack first
+    assert!(client.process(Some(pkt), now()).dgram().is_some());
+    // We do not have a ResumptionToken evennt yet, because NEW_TOKEN was not sent.
+    assert_eq!(get_tokens(&mut client).len(), 0);
+    // process() will return a Callback that has a value of the resumption token timer.
+    let callback = client.process(None, now());
```

Comment:
This would seem to depend on the resumption token timer.  If you would prefer to keep the extra timer to keep testing easy, then it's not bad that you have the extra code there.

(We should use an iterator/generator though...)